src/test/kotlin/org/malachite/estella/application/ApplicationIntegration.kt

```diff
@@ -0,0 +1,38 @@
+package org.malachite.estella.application
+
+import org.junit.jupiter.api.MethodOrderer
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestMethodOrder
+import org.malachite.estella.BaseIntegration
+import org.malachite.estella.commons.loader.FakeLoader
+import org.malachite.estella.services.ApplicationService
+import org.malachite.estella.util.offersWithProcess
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpMethod
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+class ApplicationIntegration: BaseIntegration() {
+
+    @Autowired
+    private lateinit var applicationService: ApplicationService
```

Comment:
Po co Ci tu applicationService? Właśnie po to są te zapytania http, żeby nie musieć żadnego service'a mieć w środku (może oprócz security, żeby hasło zmienić)